docs/reactions.md

```diff
@@ -107,7 +107,7 @@ Once we run the `for` loop to change the `energyLevel` with the `reduceEnergy`
 action, we see a new log entry every time an `autorun` function observes a
 change in its observable state:
 
-1.  For the _"Energy level"_ function, this is every time the `energyLevel` observable changes, 10 times in total.
+1.  For the _"Energy level"_ function, this is every time the `energyLevel` observable changes, 11 times in total.
```

Comment:
The number clearly refers to the number of changes:

> **Once we run the `for` loop to change** the `energyLevel` with the `reduceEnergy`
action, we see a **new log entry every time an `autorun` function observes a
change** in its observable state:
> 1.  For the _"Energy level"_ function, this is **every time the `energyLevel` observable changes**, 10 times in total.

Reply:
Ok, I got a little confused there. Closing the PR.